### PR TITLE
Allow bootstrapping of desktop and CoreCLR

### PR DESCRIPTION
**Pull request by jeffkl**
This sets the bootstrap destination as "Bootstrap-NetCore" for CoreCLR and improves the logic to delete the folder prior to bootstrapping.

I build both all of the time but the CoreCLR bootstrapping wipes out the desktop one so I can't use them both!

### CODE REVIEW

**Comment by jeffkl** [comment]
@dotnet-bot test Windows_NT Build for CoreCLR please

File: targets/BootStrapAppLocalMSBuild.proj [hunk 1]
```diff
@@ -11,7 +11,12 @@
     </ItemGroup>
 
     <RemoveDir
-            Directories="$(BootstrapDestination)" />
+            Directories="$(BootstrapDestination)"
+            ContinueOnError="true"/>
+
+    <RemoveDir
+            Directories="$(BootstrapDestination)"
+            ContinueOnError="true"/>
```
Comment (rainersigwald) [inline targets/BootStrapAppLocalMSBuild.proj]: Duplicated? For a reason?

File: dir.props [hunk 2]
```diff
@@ -205,6 +199,15 @@
     <PlatformTarget Condition="'$(PlatformSpecificBuild)' == 'true' AND '$(FullFrameworkBuild)' != 'true'">AnyCPU</PlatformTarget>
   </PropertyGroup>
 
+  <!-- Configure bootstrapping -->
+  <PropertyGroup>
+    <BootstrapDestination Condition="'$(NetCoreBuild)' != 'true'">$(BinDir)Bootstrap\</BootstrapDestination>
```
Comment (cdmihai) [inline dir.props]: Make sure that !windows are not creating directories with '\' in their name :)

File: targets/BootStrapAppLocalMSBuild.proj (hunk 1 above)
Comment (jeffkl) [inline targets/BootStrapAppLocalMSBuild.proj]: I'll remove the dupes for now

File: dir.props (hunk 2 above)
Comment (jeffkl) [inline dir.props]: Good point, I'll bring back the PathSeparator character instead.

Comment (rainersigwald) [inline dir.props]: This inspired #1302.

File: dir.props [hunk 3]
```diff
@@ -205,6 +199,15 @@
     <PlatformTarget Condition="'$(PlatformSpecificBuild)' == 'true' AND '$(FullFrameworkBuild)' != 'true'">AnyCPU</PlatformTarget>
   </PropertyGroup>
 
+  <!-- Configure bootstrapping -->
+  <PropertyGroup>
+    <BootstrapDestination Condition="'$(NetCoreBuild)' != 'true'">$(BinDir)Bootstrap$([System.IO.Path]::DirectorySeparatorChar)</BootstrapDestination>
```
Comment (jeffkl) [inline dir.props]: @cdmihai here is where I appended the DirectorySeparatorChar as you requested

**Pull request merged**